ansible/memcached-operator/molecule/test-local/molecule.yml

```diff
@@ -6,21 +6,26 @@ driver:
 lint:
   name: yamllint
   enabled: False
+  options:
+    config-data:
+      line-length:
+        max: 120
+
 platforms:
-- name: kind-test-local
-  groups:
-  - k8s
-  image: bsycorp/kind:latest-1.15
-  privileged: True
-  override_command: no
-  exposed_ports:
-    - 8443/tcp
-    - 10080/tcp
-  published_ports:
-    - 0.0.0.0:${TEST_CLUSTER_PORT:-10443}:8443/tcp
-  pre_build_image: yes
-  volumes:
-    - ${MOLECULE_PROJECT_DIRECTORY}:/build:Z
+  - name: kind-test-local
```

Comment:
@fabianvf all these changes scaffold X samples in the molecule are not in the migration guide. 
Did you make them in this release or it was outdated? Do you know it?

Reply:
None of the changes to the molecule scenarios are strictly necessary, the new stuff is just cleaner, though I should probably add a section touching on the bigger changes

Reply:
I think we can add something in the migration doc to share the changes anyway. We usually put (optional) and describe how to get that. However, really tks for letting me know and your review.